.github/dependabot.yml

```diff
@@ -22,3 +22,7 @@ updates:
     directory: "/internal/tools"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/cmd/checkdoc"
+    schedule:
+      interval: "weekly"
```

Comment:
Let's for the moment leave it in the main module, since we agreed to not have too many modules.

Reply:
If we need to run CheckDocs binary in `contrib` workflow, I am not sure how to get it installed along there if we leave it in main module.

Reply:
You can install a specific command from a module, I think :)

Reply:
We can do that later